tracer/src/Datadog.Trace/AppSec/EventTrackingSdk.cs

```diff
@@ -98,9 +98,9 @@ internal static void TrackUserLoginFailureEvent(string userId, bool exists, IDic
 
             var setTag = TaggingUtils.GetSpanSetter(span);
 
-            setTag(Tags.AppSec.EventsUsersLogin.FailureTrack, bool.TrueString);
+            setTag(Tags.AppSec.EventsUsersLogin.FailureTrack, "true");
             setTag(Tags.AppSec.EventsUsersLogin.FailureUserId, userId);
-            setTag(Tags.AppSec.EventsUsersLogin.FailureUserExists, exists.ToString());
+            setTag(Tags.AppSec.EventsUsersLogin.FailureUserExists, exists.ToString().ToLower());
```

Comment:
Lets avoid being so allocatey 😄 
```suggestion
            setTag(Tags.AppSec.EventsUsersLogin.FailureUserExists, exists ? "true" : "false");
```